dicomexporter/exporter.py

```diff
@@ -170,7 +176,10 @@ def convertDICOMVolumeToVTKFile(
     else: # vtkjs
         writer = vtk.vtkJSONDataSetWriter()
 
-    writer.SetFileName(output_file_path)
+    if isinstance(writer, vtk.vtkJSONDataSetWriter) and get_vtk_major_version() > 9:
+        writer.GetArchiver().SetArchiveName(output_file_path)
+    else:
+        writer.SetFileName(output_file_path)

```

Comment:
Why don't you always do `writer.GetArchiver().SetArchiveName(output_file_path)` ? isn't it supported in VTK < 9 ?

Reply:
I made a mistake.
This writer doesn't exist in a version anterior than 9. 
It has been introduced in vtk 9.0.0. And since vtk 9.1.0, the `SetFileName` is deprecated.
But indeed, we totally use the archiver. It's always available since the writer exists.